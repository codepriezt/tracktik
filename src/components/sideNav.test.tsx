import React from "react"
import {shallow } from "enzyme"
import SideNav from "./sideNav"

describe('<sideNav/>', () => {
    it('render side menu', () => {
        expect(shallow(<SideNav />)).toMatchSnapshot()
    })
})


