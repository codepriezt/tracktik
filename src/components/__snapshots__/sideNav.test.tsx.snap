// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<sideNav/> render side menu 1`] = `<Fragment />`;
